_sass/_partials--service-go.scss

```diff
@@ -0,0 +1,35 @@
+$dimension--service-go-bullet:                    8px;
+
+.service--go {
+  .service-section {
+    &--alt {
+      background-color: $radical-red;
+    }
+  }
+
+  .service-lead {
+    background-color: $martinique;
+  }
+
+  .service-how-list-item {
```

Comment:
This should be on `_objects--lists.scss` cause I will suppose it is used on all the site, for all bullet list I think, and then on each service partial we set a custom icon